modules.json

```diff
@@ -10,14 +10,14 @@
                         "git_sha": "5e34754d42cd2d5d248ca8673c0a53cdf5624905",
                         "installed_by": ["modules"]
                     },
-                    "bcftools/mpileup": {
+                    "bcftools/concat": {
```

Comment:
Just changed the order, right?

Reply:
yes, it was all done by nf-core/tools, I was super worried, but @mashehu saved the day by explaining it was just a re-order